internal/integration/sqlite_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"ariga.io/atlas/sql/schema"
 	"ariga.io/atlas/sql/sqlite"
 
+	_ "github.com/libsql/libsql-client-go/libsql"
```

Comment:
Hey! Why are the tests are here? We agreed the tests will reside on a separate repo?